locations/perleberger36.yml

```diff
@@ -88,16 +125,30 @@ networks:
     dns: 1
     ipv6_subprefix: 1
     assignments:
+      # 10.36.243.129/32
       perleberger36-core: 1
+      # 10.36.243.130/32
       perleberger36-dachboden-erx1: 2
-      perleberger36-dachboden-erx2: 2
-      perleberger36-turm-erx1: 2
-      perleberger36-turm-erx2: 2
-      perleberger36-turm-2ghz-nw: 2
-      perleberger36-turm-2ghz-no: 2
-      perleberger36-turm-2ghz-o: 2
-      perleberger36-turm-2ghz-s: 2
-      perleberger36-turm-2ghz-so: 2
-      perleberger36-turm-2ghz-sw: 2
-      perleberger36-hof-2ghz-n: 2
-      perleberger36-hof-2ghz-s: 2
+      # 10.36.243.131/32
+      perleberger36-dachboden-erx2: 3
+      # 10.36.243.132/32
+      perleberger36-turm-erx1: 4
+      # 10.36.243.133/32
+      perleberger36-turm-erx2: 5
+      # 10.36.243.134/32
+      perleberger36-turm-2ghz-nw: 6
+      # 10.36.243.135/32
+      perleberger36-turm-2ghz-no: 7
+      # 10.36.243.136/32
+      perleberger36-turm-2ghz-o: 8
+      # 10.36.243.137/32
+      perleberger36-turm-2ghz-so: 9
+      # 10.36.243.138/32
+      perleberger36-turm-2ghz-sw: 10
+      # 10.36.243.139/32
+      # 10.36.243.140/32
+      perleberger36-hof-2ghz-s: 12
+      # 10.36.243.141/32
+      # offline devices / association unknown
+      # perleberger36-turm-2ghz-s: 1
+      # perleberger36-hof-2ghz-n: 1
```

Comment:
There is no turm-2ghz-n

Reply:
It's `hof-2ghz-n` and the Wiki references that device: https://wiki.freifunk.net/Berlin:Standorte:Heilige-Geist-Kirche_Moabit